src/main/kotlin/com/terraformation/backend/accelerator/db/ParticipantProjectSpeciesStore.kt

```diff
@@ -78,6 +79,31 @@ class ParticipantProjectSpeciesStore(
     }
   }
 
+  fun deleteMany(participantProjectSpeciesIds: Set<ParticipantProjectSpeciesId>) {
```

Comment:
Bikeshedding a bit, and feel free to ignore this if you like the current name better: this could be named `delete`. There is no single-ID delete method and the fact that this takes a collection argument already documents that it can take more than one ID.